clusterman/batch/cluster_metrics_collector.py

```diff
@@ -102,15 +102,17 @@ def configure_initial(self) -> None:
             self.pools[scheduler] = get_pool_name_list(self.options.cluster, scheduler)
         for scheduler, pools in self.pools.items():
             for pool in pools:
-                self.config.watchers.append({
-                    f'{pool}.{scheduler}': get_pool_config_path(self.options.cluster, pool, scheduler),
-                })
+                watcher = {
+                    f'{pool}.{scheduler}': get_pool_config_path(self.options.cluster, pool, scheduler)
+                }
+                self.add_watcher(watcher)
                 load_cluster_pool_config(self.options.cluster, pool, scheduler, None)
+        self.initialize_clusterman_metrics_client()
 
+    def initialize_clusterman_metrics_client(self):
         self.region = staticconf.read_string('aws.region')
         self.run_interval = staticconf.read_int('batches.cluster_metrics.run_interval_seconds')
         self.logger = logger
```

Comment:
You should probably move this line out of the function. Doesn't seem like it depends on srv-configs at all.

Reply:
Could you clarify on this?